integration-tests/utils/Login.ts

```diff
@@ -30,6 +30,10 @@ export class Login {
   private static waitForApps() {
     Common.waitForLoad();
     Common.verifyPageTitle(pageTitles.overviewPage);
+    cy.get('[data-test="view-my-applications"]')
+      .should('be.visible')
+      .and('have.attr', 'aria-disabled')
```

Comment:
I guess `.should('be.enabled')` should also work.

Reply:
Do you mean instead of checking `aria-disabled=false` I should just use this `be.enabled` check?

Reply:
I tried locally: 
```
    cy.get('[data-test="view-my-applications"]')
      .should('be.visible')
      .and('be.enabled')
```
and also
```
    cy.get('[data-test="view-my-applications"]')
      .should('be.enabled')
```
but failed with: 
```
Timed out retrying after 40000ms: expected '<a.pf-c-button.pf-m-secondary.pf-m-display-lg.intro-banner__cta>' to be 'enabled'
```
Not sure why it is checking this button instead of the element I get by `cy.get()`.

Reply:
its checking  <a> tag only not button as per the above failure.